packages/visual-dev/src/components/stories/Switch.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Switch } from '../Switch'
+
+export default {
+	title: "Components / Switch",
+	component: Switch
+}
+
+export const Default = () => <Switch id="default"></Switch>
+export const On = () => <Switch status="on" id="on"></Switch>
+export const Critical = () => <Switch status="on" color="critical" id="critical"></Switch>
+export const Off = () => <Switch status="off" id="off"></Switch>
```

Comment:
Just for completeness I might add one more story where the switch is critical and on by default.

Reply:
Did you mean off by default? Critical story is a switch where it is critical and it's status set as 'on' on line 11 above there.

Reply:
Sorry yeah off by default